src/command/run.rs

```diff
@@ -36,11 +36,11 @@ pub fn run(target: Target, in_file: PathBuf) -> Result<(), String> {
                 Ok(process) => process,
             };
 
-            &process
+            process
                 .stdin
                 .unwrap()
                 .write_all(&buf)
-                .map(|_| "Could not write to nodejs process".to_string());
+                .expect("Could not write to nodejs process");
```

Comment:
Maybe write a message to stderr followed by `process::exit(1)` instead of panicing?

Reply:
Good point👍